### ISSUE TITLE
Use HTTPS

### ISSUE THREAD

**Issue by teravest** [qiime/qiime_web_app#618]
We really should...

**Comment by ElDeveloper** [comment]
:+1:

2014-02-11 15:53 GMT-07:00 teravest notifications@github.com:

**Comment by adamrp** [comment]
Yep!  Do we need a new certificate for this, or can we use an existing one?
 Probably a subject for the upcoming systems meeting.
